opnode/service.go

```diff
@@ -80,3 +80,19 @@ func NewLogConfig(ctx *cli.Context) (node.LogConfig, error) {
 	}
 	return cfg, nil
 }
+
+func NewSnapshotLogger(ctx *cli.Context) (log.Logger, error) {
+	snapshotFile := ctx.GlobalString(flags.SnapshotLog.Name)
+	handler := log.StreamHandler(os.Stdout, log.JSONFormat())
```

Comment:
Can we not default to stdout and just no-op the logger if we don't get a file?

Reply:
Done.